litmus-portal/frontend/src/views/CreateWorkflow/ReliabilityScore/__tests__/ReliabilityScore.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { render } from '../../../../testHelpers/test-util';
+import ReliabilityScore from '../index';
+import '@testing-library/jest-dom';
+
+describe('ReliabilityScore', () => {
+  it('Renders', () => {
+    render(<ReliabilityScore />);
+
+    const sliders = screen.getAllByRole('slider');
+    sliders.forEach((slider) => {
+      expect(slider).toHaveTextContent('10');
```

Comment:
can we add an user event test here? @saswatamcode

Reply:
Sure!